packages/create-sitecore-jss/src/templates/nextjs-sxa/src/Layout.tsx

```diff
@@ -47,29 +49,28 @@ const Layout = ({ layoutData, headLinks }: LayoutProps): JSX.Element => {
 
       {/* root placeholder for the app, which we add components to using route data */}
       <div className={mainClassPageEditing}>
-        <% if (templates.includes('nextjs-xmcloud') || templates.includes('nextjs-sxa')) { %>
-          {layoutData.sitecore.context.renderingType === RenderingType.Component ? (
+        <% if (templates.includes('nextjs-xmcloud')) { %>
+          {layoutData.sitecore.context.renderingType === RenderingType.Component && (
```

Comment:
We can't really do that this way, because if we are in _DesignLibrary_ we have to render **ONLY** _DesignLibrary_ component, the rest of the markup below shouldn't appear